config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -0,0 +1,75 @@
+presets:
+- labels:
+    preset-cloudprovider-azure-cred: "true"
+  env:
+  - name: AZURE_CREDENTIALS
+    value: /etc/azure-cred/credentials
+  - name: AZURE_SSH_PUBLIC_KEY_FILE
+    value: /etc/azure-ssh/azure-ssh-pub
+  - name: REGISTRY
+    value: k8sprow.azurecr.io
+  - name: KUBE_VERBOSE
+    value: 0
+  volumes:
+  - name: azure-cred
+    secret:
+      secretName: azure-cred
+  - name: azure-ssh
+    secret:
+      secretName: azure-ssh
+  volumeMounts:
+  - name: azure-cred
+    mountPath: /etc/azure-cred
+    readOnly: true
+  - name: azure-ssh
+    mountPath: /etc/azure-ssh
+    readOnly: true
+
+periodics:
+- interval: 6h
+  # ci-dualstack-azure-e2e runs IPv6DualStack feature specific tests on aks-e cluster.
+  name: ci-dualstack-azure-e2e
+  labels:
+    preset-service-account: "true"
+    preset-cloudprovider-azure-cred: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190607-b96146a3c-master
+      args:
+      - --job=$(JOB_NAME)
+      - --repo=k8s.io/kubernetes=v1.15.0
+      - --repo=k8s.io/$(REPO_NAME)=master
+      - --root=/go/src
+      - --service-account=/etc/service-account/service-account.json
+      - --timeout=450
+      - --upload=gs://kubernetes-jenkins/logs/
+      - --scenario=kubernetes_e2e
+      - --
+      - --gce-ssh=
+      - --test=true
+      - --up=true
+      - --down=true
+      - --deployment=acsengine
+      - --build=bazel
+      - --provider=skeleton
+      - --ginkgo-parallel=30
+      - --test-ccm=True
+      - --acsengine-agentpoolcount=2
+      - --acsengine-admin-username=azureuser
+      - --acsengine-creds=$AZURE_CREDENTIALS
+      - --acsengine-orchestratorRelease=1.15
+      - --acsengine-mastervmsize=Standard_DS2_v2
+      - --acsengine-agentvmsize=Standard_D4s_v3
+      - --acsengine-ccm=True
+      - --acsengine-location=eastus2
+      - --acsengine-public-key=$AZURE_SSH_PUBLIC_KEY_FILE
+      # need to update template location
+      - --acsengine-template-url=https://gist.githubusercontent.com/aramase/9431e9701a916e782406005f57d6f20d/raw/5397e4a325e0b06423d682f1de44dce770ff611f/myapimodel.json
+      # need to update this with release once aks-e PR is merged
+      - --acsengine-download-url=https://github.com/Azure/aks-engine/releases/download/v0.37.1/aks-engine-v0.37.1-linux-amd64.tar.gz
```

Comment:
Need to bump this version once all 1.16 aks-e PRs are merged

Reply:
aks-e v0.39.0 was released with 1.16.0-alpha.1 support. So I've updated the `template-url` and binary url to the correct versions.